Settings.cfg

```diff
@@ -0,0 +1,5 @@
+// Mount & Blade II Bannerlord - Dismemberment Mod by jedijosh920
+// Settings configuration file
+Chance=1
+SlowMotion=false
+Debug=true
```

Comment:
I don't see that this file is being loaded anymore. Can we delete this Settings.cfg or should we be loading it in still? (or am I missing where it's loaded?)

Reply:
reminds me of endourse jedijosh920. We have to state, that our improvement is based on his original mod.

Reply:
did this in the readme pull request